recipes/plumbum/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "plumbum" %}
+{% set version = "1.6.2" %}
+{% set sha256 = "75eff3a55e056d8fc06f7b7ceb603ce4c26650cd6a2196bcdb0b80fee59471a8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - paramiko
+
+test:
+  imports:
+    - plumbum
+    - plumbum.cli
+    - plumbum.colorlib
+    - plumbum.commands
+    - plumbum.fs
+    - plumbum.machines
+    - plumbum.path
+
+  requires:
+    - pytest
+    - paramiko
+
+about:
+  home: https://plumbum.readthedocs.io
+  license: MIT
+  license_file: LICENSE
+  summary: 'Plumbum: shell combinators library'
+
+  description: |
+    Apart from shell-like syntax and handy shortcuts, the library provides local and
+    remote command execution (over SSH), local and remote file-system paths, easy
+    working-directory and environment manipulation, a programmatic Command-Line
+    Interface (CLI) application toolkit, and a color library.
+  doc_url: https://plumbum.readthedocs.io/
+  dev_url: https://github.com/tomerfiliba/plumbum
+
+extra:
+  recipe-maintainers:
+    - henryiii
+    - tomerfiliba
```

Comment:
@tomerfiliba -- Are you ok being added as a co-maintainer on this conda-forge feedstock?